docs/source/changelog.rst

```diff
@@ -8,6 +8,9 @@ Changelog
 
 - Add :class:`pandas_gbq.Context` to cache credentials in-memory, across
   calls to ``read_gbq`` and ``to_gbq``. (:issue:`198`, :issue:`208`)
+- Fast queries now do not log above ``DEBUG`` level. (:issue:`204`).
+  With BigQuery's release of `clustering <https://cloud.google.com/bigquery/docs/clustered-tables>`__
+  querying smaller samples of data is now more faster and cheaper.
```

Comment:
Tell me if this sounds sales-y... (I'm not on commission!)

Reply:
Ha. It does a little bit, but it's nice to get the word out. :-)